packages/sdk/src/utils/tryLoadEnvs.ts

```diff
@@ -37,14 +37,14 @@ export function tryLoadEnvs(
     rootEnvPath: string | null | undefined
     schemaEnvPath: string | null | undefined
   },
-  opts: { conflictCheck: 'warn' | 'error' | 'none' } = {
+  opts: { conflictCheck: 'warn' | 'none' } = {
     conflictCheck: 'none',
   },
 ): LoadedEnv {
   const rootEnvInfo = loadEnv(rootEnvPath)
   if (opts.conflictCheck !== 'none') {
     // This will throw an error if there are conflicts
-    checkForConflicts(rootEnvInfo, schemaEnvPath, opts.conflictCheck)
+    checkForConflicts(rootEnvInfo, schemaEnvPath)
```

Comment:
Here a default value 'none' is assigned to `conflictCheck` but then the value is not passed anymore to `checkForConflicts()`.
Which means we also removed the `conflictCheck: 'none'` behavior?